packages/account/src/Sections/Security/Passkeys/components/passkey-rename.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Form, Formik } from 'formik';
+import { localize, Localize } from '@deriv/translations';
+import { DerivLightIcEditPasskeyIcon } from '@deriv/quill-icons';
+import { TCurrentManagedPasskey } from '../passkeys';
+import { getPasskeyRenameValidationSchema } from '../passkeys-configs';
+import { PasskeysStatusLayout, TPasskeysButtonOnClicks } from './passkeys-status-layout';
+import FormInputField from '../../../../Components/forms/form-fields/form-input-field';
```

Comment:
can we change this to `import { FormInputField } from 'Components/forms/form-fields';`